src/custom/dep.cpp

```diff
@@ -654,6 +660,11 @@ bool invertPaintForDarkMode(const DarkMode mode, NSVGshape* const shape, NSVGpai
             paint.type = NSVG_PAINT_COLOR;
             paint.color = 0xff191919;
             return true;
+        // Special case for LOGinstruments gradient
+        case kModeLOGinstruments:
+            paint.gradient->stops[0].color = 0xd95c5c5c; // original color from stop 1 was copied to this stop and made ~25% darker
```

Comment:
lines get too long here, please put the comments above the color changes so the code is more readable